roofit/roofitcore/inc/RooMinimizer.h

```diff
@@ -1,18 +1,12 @@
-/*****************************************************************************
- * Project: RooFit                                                           *
- * Package: RooFitCore                                                       *
- *    File: $Id$
- * Authors:                                                                  *
- *   WV, Wouter Verkerke, UC Santa Barbara, verkerke@slac.stanford.edu       *
- *   DK, David Kirkby,    UC Irvine,         dkirkby@uci.edu                 *
- *   AL, Alfio Lazzaro,   INFN Milan,        alfio.lazzaro@mi.infn.it        *
- *   PB, Patrick Bos,     NL eScience Center, p.bos@esciencecenter.nl        *
- *                                                                           *
- *                                                                           *
- * Redistribution and use in source and binary forms,                        *
- * with or without modification, are permitted according to the terms        *
- * listed in LICENSE (http://roofit.sourceforge.net/license.txt)             *
- *****************************************************************************/
+/*
+ * Project: RooFit
+ *
+ * Copyright (c) 2024, CERN
```

Comment:
Legally we can't do that without the explicit agreement of the authors. Let's not go that route.

Reply:
Ok, that's a good point! I was naively adapting the comment convention to the other (newer) header files, but that's not really the point of the PR. The point is to skip one abstraction layer between RooFit and the minimizers, such that developing on RooFit/minimizers is less intimidating. This change is something that I discussed with @lmoneta already months ago.

Reply:
I think this matter should be discussed more broadly as it has had serious implications on other projects. We can’t really do such changes even if a code owner endorsed them.